Sources/SnapshotTesting/Common/String+SpecialCharacters.swift

```diff
@@ -0,0 +1,61 @@
+//
```

Comment:
Cool to remove this header comment? We delete them in all of the other files.

Reply:
Sure!